tests/runtime/CMakeLists.txt

```diff
@@ -74,8 +73,12 @@ if(FLB_IN_LIB)
   endif()
 
   FLB_RT_TEST(FLB_OUT_CLOUDWATCH_LOGS   "out_cloudwatch.c")
-  FLB_RT_TEST(FLB_OUT_KINESIS_FIREHOSE  "out_firehose.c")
-  FLB_RT_TEST(FLB_OUT_KINESIS_STREAMS   "out_kinesis.c")
+    # These plugins works only on Linux
```

Comment:
More specifically not on Windows. Do we need to handle MacOS as well?

Reply:
You are right, i inverted the if statement.
`  if(NOT FLB_SYSTEM_WINDOWS)`

Problem on Windows, Input and Filter Plugins were not runtime tested due to `FLB_OUT_LIB=No`.
It was just a guess from my side, that tests will work on windows :)